.travis.yml

```diff
@@ -1,4 +1,4 @@
-dist: bionic
+dist: jammy
```

Comment:
With `bionic` I was getting a weird error with `apt-get update` inside Presto's Docker:

```
Problem executing scripts APT::Update::Post-Invoke-Success
```

Reply:
I'm fine with that